source/extensions/filters/http/abac/abac_filter.cc

```diff
@@ -0,0 +1,171 @@
+#include "extensions/filters/http/abac/abac_filter.h"
+
+#include "common/protobuf/protobuf.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace ABACFilter {
+
+using CelValue = google::api::expr::runtime::CelValue;
+using CelMap = google::api::expr::runtime::CelMap;
+using CelList = google::api::expr::runtime::CelList;
+
+namespace {
+
+constexpr absl::string_view kMetadata = "metadata";
```

Comment:
nit: this is not Envoy style guide for constants.

Reply:
I think it's reasonable in this case to have a special prefix.